### ISSUE TITLE
Limiting scanning for only QR code with CaptureManager

### ISSUE THREAD

**Issue by Isuru-Nanayakkara** [journeyapps/zxing-android-embedded#139]
'm using the version 3.2 of this library.

With `IntentIntegrator`, you can limit the scanning for only QR code like this.
```
IntentIntegrator integrator = new IntentIntegrator(this);
integrator.setDesiredBarcodeFormats(IntentIntegrator.QR_CODE_TYPES);
```

Is there a way to do the same if you're using `CaptureManager` instead?

**Comment by rkistner** [comment]
If you're using CaptureManager, you can keep on using IntentIntegrator in the same way. This is handled with the `initializeFromIntent` call.

Just make sure you're using the same way as CaptureActivity:
https://github.com/journeyapps/zxing-android-embedded/blob/master/zxing-android-embedded/src/com/journeyapps/barcodescanner/CaptureActivity.java

Another example:
https://github.com/journeyapps/zxing-android-embedded/blob/master/sample/src/main/java/example/zxing/ToolbarCaptureActivity.java

Just make sure you tell IntentIntegrator to use your Activity with `setCaptureActivity`:

https://github.com/journeyapps/zxing-android-embedded/blob/master/sample/src/main/java/example/zxing/MainActivity.java#L53

**Comment by Isuru-Nanayakkara** [comment]
Thank you for the prompt reply. I got it working! I'm new to Android. That's why I was struggling to get the hang of this.

**Comment by palicka** [comment]
@rkistner , Hi.
Sorry, i'm trying to achieve the same thing as @Isuru-Nanayakkara , but I don't understand your answer. I don't know how to merge it. Can you give me example please ? Thank you

**Comment by rkistner** [comment]
@palicka Please see the examples in my previous comment.
